modules/installation/pages/install-proxy-unified.adoc

```diff
@@ -3,7 +3,7 @@
 
 {susemgr} Proxy is a {suse} product within the {sle} product family.
 This section describes how to install {susemgr} Proxy from {sle} installation media.
-It assumes you have already registered the {susemgr} Proxy product with the {scc} and have a registration code.
+It assumes you already have valid organization credentials with {scc} and have obtained a registration code for your {susemgr} Proxy.
```

Comment:
Why do you need the organization credentials to register the proxy?

You just need the registration code.